docs/internals/contributing/new-contributors.txt

```diff
@@ -113,7 +113,7 @@ some advice to make your work on Django more useful and rewarding.
   Sometimes it can be scary to put your opinion out to the world and say "this
   ticket is correct" or "this patch needs work", but it's the only way the
   project moves forward. The contributions of the broad Django community
-  ultimately have a much greater impact than that of any one person. We can't
+  ultimately have a much greater impact than that of anyone person. We can't
```

Comment:
The proposed edit is incorrect.

Reply:
Sorry, I got it the wrong way. Reverting...